Determine if a host is monitored by the AWS integration or Datadog agent (#31873)
DataDog/documentation: content/en/account_management/billing/aws.md

You can optionally [send CloudWatch metrics to Datadog using CloudWatch Metric Streams and Amazon Data Firehose][8] instead of using the default API polling method. If your organization uses the CloudWatch Metric Streams with Kinesis method, AWS resource exclusion rules defined in the Datadog AWS integration page do not apply. You must manage all rules for including and excluding metric namespaces or specific metric names in the CloudWatch Metric Streams configuration for each of your AWS accounts within the AWS console.

> ## Check if a Host is Monitored by Agent or AWS

Reviewer: ```suggestion
## Check if a host is monitored by the Agent or AWS
```

> In the Infrastructure Host List:

Reviewer: ```suggestion
In the Infrastructure Host list:
```

- **Monitored by AWS Integration**

>   If a host displays only the AWS logo, or if its metrics are limited to the **aws.\*** namespace, it indicates that the host is monitored exclusively through the AWS integration.

Reviewer: ```suggestion
  If a host displays only the AWS logo, or if its metrics are limited to the `aws.\*` namespace, it indicates that the host is monitored exclusively through the AWS integration.
```

Author: Hey @brett0000FF , thank you for the suggestion. I would like the word to look like `aws.*`. If we I applied your suggestion, it would be `aws.\*`.  No metric looks like `aws.\*`, so I tend to not apply the change.

>   {{< img src="account_management/billing/infra-aws.png" alt="host monitored by AWS integration" >}}

Reviewer: ```suggestion
  {{< img src="account_management/billing/infra-aws.png" alt="Infrastructure Host list showing multiple hosts with only the AWS logo, indicating monitoring through the AWS integration." >}}
```

- **Monitored by the Datadog Agent**

>   If a host displays the Datadog Agent logo but not the AWS logo, or if its metrics are collected from the Datadog Agent (such as **datadog.\***, **system.\***, etc.), it indicates that the host is being monitored by the Datadog Agent.

Reviewer: ```suggestion
  If a host displays the Datadog Agent logo but not the AWS logo, or if its metrics are collected from the Datadog Agent (such as `datadog.\*`, `system.\*`, etc.), it indicates that the host is being monitored by the Datadog Agent.
```

>   {{< img src="account_management/billing/infra-agent.png" alt="host monitored by Datadog agent" >}}

Reviewer: ```suggestion
  {{< img src="account_management/billing/infra-agent.png" alt="Infrastructure Host List showing a host with the Datadog Agent logo but no AWS logo, indicating monitoring through the Datadog Agent." >}}
```
